### PR TITLE
3.7.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# 3.7.0 (2020-03-05)
+
+- Added the `inMemorySearchEnabled` and `searchApplied` properties to the list toolbar component giving consuming SKYUX components the option to allow consumers to run their own searches remotely. [#82](https://github.com/blackbaud/skyux-list-builder/pull/82)
```
Comment (Blackbaud-TrevorBurch) [inline CHANGELOG.md]: We normally have a blank line in between entries. I think we should add one here.

Comment (Blackbaud-TrevorBurch) [inline CHANGELOG.md]: It seems odd we are saying `consuming SKYUX components` here. Why aren't we just saying `giving consumers the option to run searches remotely`?

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Disregard. I misunderstood

Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: oops. fixed.

Comment (blackbaud-johnly) [reply CHANGELOG.md]: I posted in the Slack thread, but I'm about to head out, so here's my suggestion to tighten this up a little bit:

 ```Added the `inMemorySearchEnabled` and `searchApplied` properties to the list toolbar component to enable consuming SKY UX components to run searches remotely.```

Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: Done.

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-list-builder?_sp.spa=pr-3.7.0-release](https://host.nxt.blackbaud.com/skyux-list-builder?_sp.spa=pr-3.7.0-release)

**Pull request merged**